knowledge-base/grid-edit-on-row-double-click.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Edit a row on click or double-click with a popup form
+description: How to edit a row on click or double-click with a popup form
+type: how-to
+page_title: Edit a row on click or double-click with a popup form
+slug: grid-edit-row-click
+position: 
+tags: 
+ticketid: 1443681
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I would like to edit a row in the Grid when the user clicks or double-clicks on it, instead of using the command buttons.
```

Comment:
May be include "popup form" in the description as well.

Reply:
Does not have to be popup mode only, should work with inline too, and it is more commonly used with inline.